### DISCUSSION TITLE
Accessibility overview summary comments?

### DISCUSSION THREAD

**Discussion by jimhefferon** [latex3/tagging-project#601] [General]
Hello,  Recently Boris wrote the TUG board about finding a very misleading statement about TeX and LaTeX accessibility in an at least semi-official place online.  He wrote them, and I hope helped some folks learn better.  I have also had similar experiences.  

Prompted by his experience I thought to post a page with a very high level overview that people could be pointed to.  The goals are to be non-technical, to link to more technical materials, and to be short (and to get updated at least every year).  That is, I'd like that a knowledgeable person who comes across the page could use it as a starting point.  But I'd mainly like that someone who doesn't think often about this kind of stuff (such as a generic mathematician) could be induced to read it and get that good things are happening.  

I've put a draft at https://tug.org/twg/accessibility/overview.html and I'd be very grateful for any corrections, etc.  It is not currently linked to anything in the document tree because I though it would be better to first ask here for comments.

Regards,
Jim Hefferon

**Comment by hpvd** [comment]
Maybe also directly linking to the prototype (wording?-> blueprint ? ) would enable people to start giving it a try - instead of only reading about https://latex3.github.io/tagging-project/documentation/prototype-usage-instructions.html ?
